website/blog/2021-06-29-release-apache-apisix-2.7-en.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Release Apache APISIX 2.7.0"
+author: Serendipity96
+authorURL: "https://github.com/Serendipity96"
+authorImageURL: "https://avatars.githubusercontent.com/u/23514812?v=4"
+---
+> [@Serendipity96](https://github.com/Serendipity96), Apache APISIX contributor from [Shenzhen Zhiliu Technology Co.](https://www.apiseven.com/)
+>
+<!--truncate-->
+
+Apache APISIX 2.7.0 has been released! Welcome to download and use it.
+
+This version supports multi-language plugins, enhances the four-layer TCP proxy, and enhances Nginx configuration. More than 20 developers participated, and more than 70 pull requests have been submitted. The following is an introduction to the key features.
+
+## Release Notes
+
+### Multi-language plugin
+
+With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function,  the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still worried that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```

Comment:
```suggestion
With the release of the first version of apisix-java-plugin-runner, and apisix-go-plugin-runner completes its main functions, the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still worried that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```

Reply:
```suggestion
With the release of the first version of apisix-java-plugin-runner,  and apisix-go-plugin-runner completes main function, the multi-language plugin of Apache APISIX supports the two most widely used back-end programming languages. If you are still think that the development of Apache APISIX plugin will be limited to Lua ecosystem, please try our multi-language plugin runner to develop Java / Go plugins.
```

Reply:
worried seems to be an out of context word here IMO

Reply:
`worried` is fine... The original Chinese text uses the same word.

Reply:
```suggestion
With the first release of apisix-java-plugin-runner, and the completion of apisix-go-plugin-runner's main functions, the multi-language plugin of Apache APISIX now supports the two most widely used back-end programming languages. If you are still worried that the plugin development of Apache APISIX will be limited to Lua ecosystem, welcome to use our multi-language plugin runner to develop your own Java / Go plugins!
```

Reply:
I guess "hesitant" should be the right word choice here, regardless of language @spacewander as in "If you are still **hesitant** that the plugin development of Apache APISIX will be limited to Lua ecosystem". What do you think?

Reply:
Using "hesitant" is a good choice. But I think being strict with the original text is better.

Reply:
Okay 👍